test/cppsuite/test_harness/api_const.h

```diff
@@ -1,3 +1,31 @@
+/*-
```

Comment:
Is this copyright text correct for the files within test_harness/ ?

Reply:
Looks fine to me, test/format has the same copyright.

Reply:
Yes, this is okay.